new/LIT7103PrayOTprophets.xml

```diff
@@ -0,0 +1,112 @@
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient12859"/>
+                    <witness corresp="BNFabb107"/>
+                    <witness corresp="BNFabb270"/>
+                    <witness corresp="BNFeth320"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Collection of prayers of the prophets of the Old Testament, each of which are introduced with 
+                    <foreign xml:lang="gez">ወሶበ፡ ጸርሐ፡</foreign> (engl. "When he cried") or 
+                    <foreign xml:lang="gez">ወሶበ፡ ጸርሑ፡</foreign> (engl. "When they cried") and is often continued with 
+                    <foreign xml:lang="gez">ስመከ፡</foreign> (engl. "your name"), possibly related to 
+                    <ref type="work" corresp="LIT1828Mahale"/>, <ref type="work" corresp="LIT2277salotu"/>,
+                    <ref type="work" corresp="LIT2265salota"/> or <ref type="work" corresp="LIT2250salota"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="OldTestament"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-10-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <bibl><ptr target="bm:Strelcyn1949PriereFalacha"/><citedRange unit="page">64-75</citedRange></bibl>
+            </div>
+            <div type="edition">
+                <note>Incipits and order of the prayers are taken from <ref type="mss" corresp="BLorient12859"/></note> 
+                <div type="textpart" xml:id="Moses">
```

Comment:
Bible contains three Songs of Moses, here  LIT2277salotu, LIT2278salota, LIT2703salota